Xamarin.Forms.Platform.Android/Renderers/ListViewRenderer.cs

```diff
@@ -418,18 +421,28 @@ void UpdateSelectionMode()
 
 		void UpdateHorizontalScrollBarVisibility()
 		{
-			var horizontalScrollBarVisibility = Element.HorizontalScrollBarVisibility;
+			if (_defaultHorizontalScrollVisibility == ScrollBarVisibility.Default)
+				_defaultHorizontalScrollVisibility = Control.HorizontalScrollBarEnabled ? ScrollBarVisibility.Always : ScrollBarVisibility.Never;
```

Comment:
The "default" value should be whatever value the control has set if Forms doesn't intervene at all. So something like:
```
bool? _defaultHorizontalScrollBarVisibility;

void UpdateHorizontalScrollBarVisibility(){
	
	if(_defaultHorizontalScrollBarVisibility == null){
		// Runs the first time the method is called, caches the default
		_defaultHorizontalScrollBarVisibility = Control.HorizontalScrollBarEnabled;
	}
	
	switch(Element.HorizontalScrollBarVisibility) {
		case(Always):
			Control.HorizontalScrollBarEnabled = true;
			break;
		case(Never):
			Control.HorizontalScrollBarEnabled = false;
			break;
		case(Default):
			Control.HorizontalScrollBarEnabled = _defaultHorizontalScrollBarVisibility;
		break;
	}
}
```

Reply:
Ah, gotcha